dockstore-webservice/src/main/java/io/dockstore/webservice/languages/LanguageHandlerInterface.java

```diff
@@ -324,6 +333,92 @@ default String getURLFromEntry(String dockerEntry, ToolDAO toolDAO) {
         return url;
     }
 
+    // TODO: Implement for DockerHub, then gitlab and seven bridges;
+    default Set<Image> getImagesFromRegistry(String toolsJSONTable) {
+        ArrayList<Map<String, String>> dockerTools = new ArrayList<>();
```

Comment:
Can use `List<Map<String, String>>` 
https://stackoverflow.com/questions/9852831/polymorphism-why-use-list-list-new-arraylist-instead-of-arraylist-list-n kind of idea